Sources/StripeKit/Billing/Invoices/Invoice.swift

```diff
@@ -318,6 +322,15 @@ public struct Invoice: Codable {
     }
 }
 
+public struct SubscriptionDetails: Codable {
+	/// Set of key-value pairs that you can attach to an object. This can be useful for storing additional information about the object in a structured format.
+	public var metadata: [String: String]?
+	
+	public init(metadata: [String: String]?) {
```

Comment:
@Maxim-Inv One last thing, add a default nil param for this and it should be good to merge and tag anew release

Reply:
Ok. Ready.